packages/core/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -128,25 +132,29 @@ Color.story = {
   name: 'color',
 }
 
-export const ForwardRefs = () => (
-  <ForwardRefDemo
-    refChild={(dsRef) => (
+export function ForwardRefs() {
+  function refChild(dsRef) {
+    function onClick() {
+      return dsRef.current.focus()
+    }
+    return (
       <>
         {/*
-            This example is for SC3
-              <Button dsRef={e => this.btnRef = e}>Click me</Button>
-          */}
+          This example is for SC3
+            <Button dsRef={e => this.btnRef = e}>Click me</Button>
+        */}
```

Comment:
```suggestion
```

Reply:
Going to remove all forward ref demos later as discussed.